libraries/libgraphic/png/PngReader.h

```diff
@@ -13,12 +18,73 @@ class PngReader
         return _valid;
     }
 
+    inline uint32_t width()
+    {
+        return _width;
+    }
+
+    inline uint32_t height()
+    {
+        return _height;
+    }
+
+    inline const Vector<Color> &pixels()
+    {
+        return _pixels;
+    }
+
+    inline const DateTime &modified()
+    {
+        return _modified;
+    }
+
 private:
```

Comment:
Look like I never told you that I prefer the order of declarations to be private then public to public then private ;)

Reply:
wat? :D